### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by sdavidsson90**
Finished Putty.md. Added the images found in the repo already.

### CODE REVIEW

**Comment by ThomasA** [comment]
Clear and helpful, great!  
I tweaked the Markdown syntax a bit to the listings get formatted better.

File: docs/guides/openstack_guides/putty.md [hunk 1]
```diff
@@ -1,72 +1,45 @@
...
-![Alt Description](../img/openstack/puttykeygenerator.png?raw=true "Title")
-
-* Browse to the location of the private key file that you want to convert (e.g., MyPersonalKey.pem).
-
-Note: By default, PuTTYgen displays only files with extension .ppk; you'll need to change that to display files of all types in order to see your .pem key file. The private key file must end with a newline character or PuTTYgen cannot load it correctly.
-
-![Alt Description](../img/openstack/puttysavekey.png?raw=true "Title")
-
+# Connect to instance from Windows (using PuTTY) 
+Some versions of Windows do not support OpenSSH out of the box. This means that you can't use the SSH-key generated by the Openstack-platform (in the format *.pem). We recommend a solution where you use [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) to convert your SSH-key to a Windows supported format and then connect to your instance.
```
Comment (ThomasA) [inline docs/guides/openstack_guides/putty.md:2]: Isn't it more of a question of having the key in a PuTTY-supported format than a Windows-supported format? If so, this "Windows format" term is mentioned once more further down the page.

File: docs/guides/openstack_guides/putty.md [hunk 2]
```diff
@@ -1,72 +1,45 @@
...
 * Save your yourPersonalKey.ppk file somewhere secure.
 
 
-
-# SSH to instance
-
-First, we need to convert yourPersonalKey.pem to the format used with Putty before we can use PuTTY. The steps are:
-
+## Connect to instance
+Now that you have converted your SSH-key from the *.pem*-format to the *.ppk*-format supported by Windows, we can now use PuTTY to connect to our instance.
```
Comment (ThomasA) [inline docs/guides/openstack_guides/putty.md:24]: ...supported by PuTTY?

**Review by ThomasA — APPROVED** [review]

File: docs/guides/openstack_guides/putty.md (hunk 1 above)
Comment (sdavidsson90) [reply docs/guides/openstack_guides/putty.md]: Det kan du sådan set have ret i.

**Pull request merged**